### PR TITLE
Homework3

### PR DESCRIPTION
**Pull request by ermekbaeva**
Homework 3

### CODE REVIEW

File: src/test/java/com/epam/tc/hw3/pages/HomePage.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    }
+
+    public WebElement getFrameWithButton() {
+        return frameWithButton;
+    }
+
+    public boolean isNavigationBarElementsDisplayed() {
+        Boolean isElementsDisplayed = false;
+        for (WebElement element : navigationNavBarElements
+        ) {
```
Comment (ZeroShadows) [inline src/test/java/com/epam/tc/hw3/pages/HomePage.java:43]: how do you know which element does not displayed?

File: src/test/java/com/epam/tc/hw3/pages/HomePage.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        for (WebElement element : textsOnIndexPage
+        ) {
+            isElementsDisplayed = element.isDisplayed();
+        }
+        return isElementsDisplayed;
+    }
+
+    public boolean isTextsOnIndexPageDisplayed() {
+        Boolean isElementsDisplayed = false;
+        for (WebElement element : imagesOnIndexPage
```
Comment (ZeroShadows) [inline src/test/java/com/epam/tc/hw3/pages/HomePage.java:60]: same comment as in method above

File: src/test/java/com/epam/tc/hw3/pages/HomePage.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        ) {
+            isElementsDisplayed = element.isDisplayed();
+        }
+        return isElementsDisplayed;
+    }
+
+    public boolean isItemsOnSideBarDisplayed() {
+        Boolean isElementsDisplayed = false;
+        for (WebElement element : itemsOnSideBar
```
Comment (ZeroShadows) [inline src/test/java/com/epam/tc/hw3/pages/HomePage.java:69]: same comment as in method above

**Review by ZeroShadows — CHANGES_REQUESTED** [review]

File: src/test/java/com/epam/tc/hw3/pages/HomePage.java (hunk 1 above)
Comment (ermekbaeva) [reply src/test/java/com/epam/tc/hw3/pages/HomePage.java]: checks for each element was done

File: src/test/java/com/epam/tc/hw3/pages/HomePage.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        for (WebElement element : navigationNavBarElements
+        ) {
+            isElementsDisplayed = element.isDisplayed();
+        }
+        return isElementsDisplayed;
+    }
+
+    public boolean isImagesOnIndexPageDisplayed() {
+        Boolean isElementsDisplayed = false;
+        for (WebElement element : textsOnIndexPage
```
[the remark this answers is not in the document]
Comment (ermekbaeva) [reply src/test/java/com/epam/tc/hw3/pages/HomePage.java]: checks for each element was done

File: src/test/java/com/epam/tc/hw3/pages/HomePage.java (hunk 2 above)
Comment (ermekbaeva) [reply src/test/java/com/epam/tc/hw3/pages/HomePage.java]: checks for each element was done

File: src/test/java/com/epam/tc/hw3/pages/HomePage.java (hunk 3 above)
Comment (ermekbaeva) [reply src/test/java/com/epam/tc/hw3/pages/HomePage.java]: checks for each element was done

**Review by ZeroShadows — APPROVED** [review]

**Pull request merged**